### PR TITLE
Refactored searchbar to use label again for accessibility and fixed mobile support 

### PR DESCRIPTION
**Pull request by ztolley**

### CODE REVIEW

File: javascripts/modules/edit.js [hunk 1]
```diff
@@ -9,7 +9,6 @@ exports.Edit = {
     this.cacheEls();
     this.bindEvents();
     this.showDataView();
-    console.log('done');
```
Comment (teneightfive) [inline javascripts/modules/edit.js]: How did we let this slip through last time 😳

Comment (ztolley) [inline javascripts/modules/edit.js]: yeah you need to learn to code review better ;)

File: javascripts/modules/searchbar.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+/* globals require, exports */
+'use strict';
+
+const $ = require('jquery');
+
+exports.SearchBar = {
+
+  el: '.searchbar',
```
Comment (teneightfive) [inline javascripts/modules/searchbar.js]: Let's keep this consistent and use a `.js-` prefix for javascript triggers

File: javascripts/modules/searchbar.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  init: function() {
+    this.cacheEls();
+    this.bindEvents();
+
+    this.inputBlurHandler();
+  },
+
+  cacheEls: function() {
+    var element = $(this.el);
+    this.input = element.find('.searchbar__input');
```
Comment (teneightfive) [inline javascripts/modules/searchbar.js]: It can be good to use a `$` symbol to prefix elements so it's clear what they are when re-using.

Comment (ztolley) [inline javascripts/modules/searchbar.js]: not a fan of that tbh

**Pull request merged**